src/main/java/strategy/operationhandlers/BalanceOperationHandlerImpl.java

```diff
@@ -0,0 +1,9 @@
+package strategy.operationhandlers;
+
+public class BalanceOperationHandlerImpl implements OperationHandler {
+
+    @Override
+    public int getOperationQuantity(int inputQuantity) {
+        return inputQuantity;
+    }
```

Comment:
```suggestion
public class BalanceOperationHandlerImpl implements OperationHandler {
    @Override
    public void getOperationQuantity(FruitTransaction transaction) {
        Storage.storage.put(transaction.getFruit(), transaction.getQuantity());
    }
```
Lets create a Storage class, that simulates db, 
it is supposed to have Map<String, Integer> storage
String - fruit name
Integer - quantity

Handlers should accept transaction and change the storage
change each handler accordingly